roles/usegalaxy-eu.rsync-to-nfs/tasks/main.yml

```diff
@@ -3,6 +3,10 @@
   copy:
     content: |
         #!/bin/bash
+        if [ "$(whoami)" != "galaxy" ]; then
+          exec sudo -u galaxy bash "$0" "$@"
```

Comment:
isn't that unexpected for a user? 

Can we maybe instead just run the rsync command as `galaxy`user?

Reply:
From my understanding this checks if the user is *not* `galaxy` and if `true` runs the whole script as galaxy user and exits the old invocation.

When I test the same but with `root` instead of galaxy on my laptop (and an `echo $(whoami)`), the user stays the same in my shell session. just changes for the time of the script execution:
 
![image](https://github.com/user-attachments/assets/5af7d79b-4e3f-4cfb-a56b-96466f211834)